### PR TITLE
implement basic history section component

### PR DESCRIPTION
**Pull request by Stephboss9**

### CODE REVIEW

File: mm-api/models/recap.js [hunk 1]
```diff
@@ -21,63 +22,67 @@ class Recap {
...
+      }
+      else if (result.rows[i].score >=60.00) {
+        result.rows[i]["actualLetterGrade"] = "D-"
+      }
+      else if (result.rows[i].score < 60.00) {
+        result.rows[i]["actualLetterGrade"] = "F"
+      }
+    
+      }
+      console.log(result.rows)
```
Comment (Rebecca-CYLi) [inline mm-api/models/recap.js:79]: found a console log

File: motivateme-ui/src/components/DashboardPage/DashboardPage.jsx [hunk 2]
```diff
@@ -29,8 +29,9 @@ export default function DashboardPage({ user, setCurrPage }) {
         setTasks(tempTasks.data.allTasks);
       }
       let { data, error } = await apiClient.getLatestGrade();
-      if (data?.latestGrade?.scoreWithPercentage) {
-        setLatestGrade(data.latestGrade.scoreWithPercentage);
+      if (data?.latestGrades) {
+        console.log(latestGrades)
```
Comment (Rebecca-CYLi) [inline motivateme-ui/src/components/DashboardPage/DashboardPage.jsx:33]: found a console log

**Review by Rebecca-CYLi — CHANGES_REQUESTED** [review]
found console logs, please fix them

**Comment by Stephboss9** [comment]
fixed!

**Review by Rebecca-CYLi — APPROVED** [review]
revisions fixed, LGTM

**Pull request merged**